programs/program-statistics.md

```diff
@@ -1,10 +1,14 @@
 # Program Statistics
 
+These are the number of successful internships per year for each program.
+
 | Year | Program                          | Internships | Total (per year) |
-| ---- | -------------------------------- | ----------- | ---------------- |
-| 2022 | LFX Mentorship (CommunityBridge) | TBD         | TBD              |
+|------|----------------------------------|-------------|------------------|
+| 2023 | LFX Mentorship                   | TBD         | TBD              |
+|      | GSoC                             | TBD         |                  |
+| 2022 | LFX Mentorship                   | TBD         | TBD              |
 |      | GSoC                             | TBD         |                  |
```

Comment:
Do you know the numbers for 2022 @nate-double-u ?

Reply:
We can find out 🙂. Related to #1000